integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -218,6 +274,62 @@ public void testMiiConfigUpdateExistJdbc() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain with a JDBC DS using model in image and having configmap
+   * in the domain.yaml before deploying the domain. After deploying the domain crd,
+   * create a new image with diff tag name and model files that define a JDBC DataSource
+   * and update the domain crd to change image name to reload the model,
+   * generate new config and initiate a rolling restart.
+   *
+   * @throws Exception if domain creation, config update or test veriofication fails
+   */
+  @Test
+  public void testMiiConfigUpdateJdbcImage() throws Exception {
```

Comment:
i have the same concern here, it seems like there are several steps and there should be assertions after each step to make sure it is working as desired. thanks

Reply:
ok. Let me move the assertions to right after the operation is done to make it look clearer

Reply:
Mark, I made changes regarding this. I added Assertions after each step, some of them are added in the helper methods, others added in the test methods themselves. Could you please take one more look see if you are ok with the changes.